appveyor.yml

```diff
@@ -45,7 +45,11 @@ build_script:
       IF [%APPVEYOR_REPO_BRANCH:~-9%]==[-releases] (
         script\build.cmd --code-sign --compress-artifacts --create-windows-installer
       ) ELSE (
-        ECHO Skipping installer and Atom build on non-release branch
+        IF [%APPVEYOR_REPO_BRANCH%]==[master] (
+          script\build.cmd --code-sign --compress-artifacts --create-windows-installer
+        ) ELSE (
+          ECHO Skipping installer and Atom build on non-release branch
+        )
```

Comment:
I suspect there's a way to streamline this `if/else` statement. If anyone has suggestions, please let me know. :bow:

The one constraint we have to keep in mind is that [this entire conditional](https://github.com/atom/atom/blob/058c13e25258c977f6a56ccdd35532441d855d7a/appveyor.yml#L44-L57) gets collapsed into a single line when it runs on AppVeyor.

Reply:
Using an intermediate variable could help remove the duplication between the two cases: https://stackoverflow.com/a/2143396